<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>empwindow</class>
 <widget class="QMainWindow" name="empwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>908</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>96</red>
        <green>195</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>96</red>
        <green>195</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>96</red>
        <green>195</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>96</red>
        <green>195</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="2">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>-1</y>
             <width>961</width>
             <height>44</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Descending salary</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ascending salary</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Sort</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="1">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="5" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_35">
               <item>
                <widget class="QLabel" name="label_31">
                 <property name="text">
                  <string>Function</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_2">
                 <item>
                  <property name="text">
                   <string>Administrator</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Destitute Manager</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Donations Manager</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Services Manager</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Events Manager</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_40">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>First name</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_34">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_43">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_firstname_4"/>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_id_4"/>
             </item>
             <item row="15" column="1">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="6" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout_42"/>
             </item>
             <item row="11" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_41">
               <item>
                <widget class="QLabel" name="label_44">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_35">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_password_4"/>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_36">
               <item>
                <widget class="QLabel" name="label_32">
                 <property name="text">
                  <string>Last name</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_23">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_lastname_4"/>
               </item>
              </layout>
             </item>
             <item row="3" column="1">
              <layout class="QGridLayout" name="gridLayout_14"/>
             </item>
             <item row="9" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_37">
               <item>
                <widget class="QLabel" name="label_40">
                 <property name="text">
                  <string>E-mail</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_24">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_contact_4"/>
               </item>
              </layout>
             </item>
             <item row="13" column="1">
              <widget class="QPushButton" name="pushButton_edit">
               <property name="text">
                <string>Update employee</string>
               </property>
              </widget>
             </item>
             <item row="12" column="1">
              <widget class="QPushButton" name="pushButton_6">
               <property name="text">
                <string>Add employee</string>
               </property>
              </widget>
             </item>
             <item row="14" column="1">
              <widget class="QPushButton" name="pushButton_clear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_39">
               <item>
                <widget class="QLabel" name="label_42">
                 <property name="text">
                  <string>Salary</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
               </item>
              </layout>
             </item>
             <item row="10" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_38">
               <item>
                <widget class="QLabel" name="label_41">
                 <property name="text">
                  <string>Login</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_32">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_login_4"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="7" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_search"/>
        </item>
        <item row="6" column="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_iddelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_logout">
            <property name="text">
             <string>Log out</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_hawdy">
            <property name="text">
             <string>Hawdy,</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_function">
            <property name="text">
             <string>Function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_name">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_salary">
            <property name="text">
             <string>salary bigger than</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>New Card ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Employee ID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_idcard"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_idempcard"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_cardadded">
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Generate PDF C.V</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>delete employee</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Add Card</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Stats</string>
       </attribute>
       <widget class="QFrame" name="horizontalFrame">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>90</y>
          <width>741</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>30</y>
          <width>291</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Statestics</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>600</y>
          <width>371</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Salary : number of employees</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionEmployees"/>
    <addaction name="actionDistitute"/>
    <addaction name="actionDonation"/>
    <addaction name="actionServices"/>
    <addaction name="actionEvents"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_the_app"/>
    <addaction name="actionAbout_the_team"/>
   </widget>
   <addaction name="menuManagement"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout_the_app"/>
   <addaction name="actionEmployees"/>
   <addaction name="actionDistitute"/>
   <addaction name="actionDonation"/>
   <addaction name="actionServices"/>
   <addaction name="actionEvents"/>
  </widget>
  <action name="actionEmployees">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/xxxxxxxxx-01.png</normaloff>:/images/xxxxxxxxx-01.png</iconset>
   </property>
   <property name="text">
    <string>Employees</string>
   </property>
  </action>
  <action name="actionDistitute">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Untitled-1-01.png</normaloff>:/images/Untitled-1-01.png</iconset>
   </property>
   <property name="text">
    <string>Distitute</string>
   </property>
  </action>
  <action name="actionDonation">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/sdf1-01.png</normaloff>:/images/sdf1-01.png</iconset>
   </property>
   <property name="text">
    <string>Donation</string>
   </property>
  </action>
  <action name="actionServices">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/xxxxxxccccccccccccc-01.png</normaloff>:/images/xxxxxxccccccccccccc-01.png</iconset>
   </property>
   <property name="text">
    <string>Services</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/qsdfg-01.png</normaloff>:/images/qsdfg-01.png</iconset>
   </property>
   <property name="text">
    <string>Events</string>
   </property>
  </action>
  <action name="actionAbout_the_app">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/logo application.png</normaloff>:/images/logo application.png</iconset>
   </property>
   <property name="text">
    <string>About the app</string>
   </property>
  </action>
  <action name="actionAbout_the_team">
   <property name="text">
    <string>About the team</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
